courses/views_test.py

```diff
@@ -414,9 +414,15 @@ def test_user_enrollment_delete(
     enrollment = CourseRunEnrollmentFactory.create(
```

Comment:
I think we should remove this instance of enrollment since we are now creating an inactive enrollment below. Let me know if it's needed anyway.

Reply:
it's being using at other places and being changed to unenroll in mitxonline but we have only mocked the edx enrollment deactivation method.